### PR TITLE
Feature: Replace cronjobs with systemd timers

### PR DESCRIPTION
**Pull request by aconitumnapellus**
##### SUMMARY
This MR replaces cron as the used scheduling software with systemd timers.

This provides better logging and scheduling of jobs and needs less hacking to prevent parallel execution of sync jobs.
Also this adds dependencies and collections needed to the meta file.

It also provides:
- Max Runtime in seconds for sync jobs
- (Potential) Logging in journald of jobs (Currently, rsync logs to a file, so you wont see much)
- Feature to easily disable and enable timers
- Feature to start a sync job manually

Additionally, support for RPM-based distributions and Ubuntu was removed, because we do not need it internally and that reduces the maintenance work.

EDIT: Molecule is running with a custom container image that contains systemd as init. Tests all run through.

##### ISSUE TYPE
 - Feature Pull Request

##### ANSIBLE VERSION
```
ansible [core 2.14.4]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: tasks/configuration.yml
Comment (s3lph) [inline tasks/configuration.yml:71]:
```suggestion
  register: systemd_analyze_output
  changed_when: false
```
As this command does not change anything, it should report as such.

**Review by s3lph — CHANGES_REQUESTED** [review]
Generally LGTM, just some minor things. Maybe @Melkor333 can have a second look at the Molecule stuff.

**Review by s3lph — APPROVED** [review]
LGTM, but as already mentioned, please get a second opinion from @Melkor333 before merging.

**Comment by Melkor333** [comment]
The verify step is a bit funny but given that this repo doesn't seem to configure something like a webserver it's probably acceptable

I don't really understand the variable definitions but the plain molecule stuff lgtm.

**Comment by aconitumnapellus** [comment]
> The verify step is a bit funny but given that this repo doesn't seem to configure something like a webserver it's probably acceptable
> 
> I don't really understand the variable definitions but the plain molecule stuff lgtm.

The verify step is the default step from molecule. I didnt implement verification because it seems rather silly. What do i need to verify? Ansible run through with the state i give it. It is already how i want.

**Review by Melkor333 — APPROVED** [review]

**Pull request merged**
